### DISCUSSION TITLE
auth_openidc_module configuration

### DISCUSSION THREAD

**Discussion by nchetbani** [OpenIDC/mod_auth_openidc#808] [Q&A]
I have configure the auth_openidc module in my apache as follows:

```
...
LoadModule auth_openidc_module /usr/lib64/httpd/modules/mod_auth_openidc.so
...
DocumentRoot "/var/www/html"
...

<VirtualHost *:80>
   ServerName myApp.com
   ...

   OIDCProviderMetadataURL https://<my.op.url>/.well-known/openid-configuration
   OIDCClientID openidc_example
   OIDCClientSecret <secret>
   **OIDCRedirectURI http://myApp.com/protected/redirect_uri**
   OIDCCryptoPassphrase <password>
   OIDCJWKSRefreshInterval 3600
   OIDCSSLValidateServer Off

   <Location />
      AuthType openid-connect
      Require valid-user
   </Location>

</VirtualHost>
```

The **html** directory contains the frontend part (html, js, css, ...) of my application. As descrobed in this article I defined a dummy URI that points to a directory protected by the module and saved it as a redirect URI in the client config on the OP

When I launch the application after deployment, I am redirected to authentication. But after authentication, I get 500 errors on loading static files nad the token call is not happening

Any idea please ?

**Comment by zandbelt** [comment]
you'll need to enable server debug logging (`LogLevel auth_openidc:debug`) and look at the log entries to find out why the 500 is returned; also, which version of the module are you using?
